src/indexmap.rs

```diff
@@ -826,6 +829,7 @@ where
     }
 }
 
+#[derive(Clone)]
```

Comment:
This should be a manual `Clone` implementation. No need for `Clone` bounds on any of the type parameters.